src/drivers/lib.rs

```diff
@@ -7,6 +7,7 @@ extern crate common;
 extern crate hil;
 extern crate process;
 
+pub mod noop;
```

Comment:
Same comment as for `yield`. Do we need this?

If so, it probably belongs somewhere else. For example, is it general enough to have it's own system call? Maybe wait can take an argument where one of them means yield? Basically, not clear what the purpose is.